src/lambdaHandler.ts

```diff
@@ -78,18 +73,22 @@ export function startServerAndCreateLambdaHandler<
     TContext
   > = options?.context ?? defaultContext;
 
-  return async function (event: any, context: any) {
-    const resultMiddlewareFns: Array<(result: any) => any> = [];
+  return async function (event, context) {
+    const resultMiddlewareFns: Array<
```

Comment:
Can this be defined in terms of the middleware types we have in `middleware.ts`?

Reply:
Good idea!